co-authors-plus.php

```diff
@@ -784,8 +792,9 @@ function coauthors_update_post( $post_id, $post ) {
 	}
 
 	function has_author_terms( $post_id ) {
-		$terms = wp_get_object_terms( $post_id, $this->coauthor_taxonomy, array( 'fields' => 'ids' ) );
-		return ! empty( $terms ) && ! is_wp_error( $terms );
+		$term_objects = get_the_terms( $post_id, $this->coauthor_taxonomy );
+		$terms = ( ! is_wp_error( $term_objects ) ) ? wp_list_pluck( $term_objects, 'term_id' ) : array();
```

Comment:
The list pluck is unnecessary here. We can just do `return ! is_wp_error( $term_objects ) && ! empty( $term_objects )`